Library/ENV/4.3/cc

```diff
@@ -1,6 +1,10 @@
-#!/System/Library/Frameworks/Ruby.framework/Versions/Current/usr/bin/ruby -W0
+#!/bin/sh
+# Make sure this shim uses the same Ruby interpreter that is used by Homebrew.
+unset RUBYLIB
+unset RUBYOPT
+exec "$HOMEBREW_RUBY_PATH" -x "$0" "$@"
```

Comment:
It is already there, but in the shebang just below this line. To quote from the Ruby man page on `-x`:

> Leading garbage will be discarded until the first line that starts with `#!` and contains the string `ruby`. Any meaningful switches on that line will be applied.